src/groupeddataframe/splitapplycombine.jl

```diff
@@ -2,6 +2,8 @@
 # groupby(), map(), combine(), by(), aggregate() and related
 #
 
+@nospecialize
```

Comment:
I'd prefer specifying this on particular methods. That's more explicit (by looking at a diff you wouldn't even notice that you're in a `@nospecialize` area).

Reply:
OK - what is the best way to do it? Do you know if writing something like:
```
function f(x,y,z)
    @nospecialize
    ...
end
```
is correct?

Reply:
The solution that was used until now is `f(@nospecialize(x), y, z)`.

Reply:
I know, but we do not want to specialize this function at all (not only for one specific argument). I guess this is what I have proposed (I will check the timings to know for sure).

Reply:
Which function?

Reply:
I have just added @nospecialize to every function separately.